src/WebJobs.Script/Workers/Rpc/RpcWorkerChannel.cs

```diff
@@ -107,10 +107,13 @@ internal class RpcWorkerChannel : IRpcWorkerChannel, IDisposable
 
         public IDictionary<string, BufferBlock<ScriptInvocationContext>> FunctionInputBuffers => _functionInputBuffers;
 
-        public RpcWorkerChannelState State => _state;
-
         internal IWorkerProcess WorkerProcess => _rpcWorkerProcess;
 
+        public bool IsChannelReadyForInvocations()
+        {
+            return _state.HasFlag(RpcWorkerChannelState.InvocationBuffersInitialized | RpcWorkerChannelState.Initialized);
```

Comment:
Should this be:
```
return _state.HasFlag(RpcWorkerChannelState.InvocationBuffersInitialized) && _state.HasFlag(RpcWorkerChannelState.Initialized);
```
like you have in test?

Reply:
they mean the same :)

Reply:
updated the test code to use similar style